xwiki-platform-core/xwiki-platform-skin/xwiki-platform-skin-skinx/src/main/java/com/xpn/xwiki/web/sx/SxDocumentSource.java

```diff
@@ -138,7 +138,8 @@ public String getContent()
                     LESSResourceReference lessResourceReference =
                         lessResourceReferenceFactory.createReferenceForXObjectProperty(objectPropertyReference);
                     try {
-                        sxContent = lessCompiler.compile(lessResourceReference, true, (parse == 1), false);
+                        sxContent = lessCompiler.compile(lessResourceReference, true, (parse == 1),
+                            CachePolicy.FORBID.equals(getCachePolicy()));
```

Comment:
It's an enum so you can use `==`.

Reply:
That too :)

Reply:
> BTW, should it be CachePolicy.FORBID != getCachePolicy() ?

Nope, the last argument is:
> * @param force force the computation, even if the output is already in the cache (not recommended)

So we want to force the computation only when `CachePolicy.FORBID == getCachePolicy()`

Reply:
Right, I looked at the wrong diff and tough `false` was your previous version :)

So indeed it should be false unless we don't want to cache